atomic_reactor/util.py

```diff
@@ -907,14 +897,16 @@ def get_inspect_for_image(
             # set Id, which isn't in config blob
             # Won't be set for v1,as for that image has to be pulled
             'Id': config_digest,
+            'Architecture': blob_config['architecture'],
+            'Os': blob_config['os'],
+            # According to OCI spec, 'created' and 'config' are optional
+            'Created': blob_config.get('created'),
+            'Config': blob_config.get('config') or {},
```

Comment:
Note that Created is not used anywhere in atomic-reactor, so even if it is None, it not break anything. Unlike Config, where some code expects it to always be there as a (possibly empty) dict.